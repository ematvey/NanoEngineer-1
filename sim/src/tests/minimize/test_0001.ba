LENGTH 0 1 0.769250284368
LENGTH 0 2 0.769803221609
LENGTH 0 3 1.53979316793
LENGTH 0 6 1.53871732297
LENGTH 9 10 0.769084520713
LENGTH 9 11 0.7700012987
LENGTH 9 12 1.73233484061
LENGTH 3 4 0.770638047335
LENGTH 3 5 0.769338027138
LENGTH 3 9 1.54048953258
LENGTH 12 13 0.770462847904
LENGTH 12 14 0.768425012607
LENGTH 6 7 0.770408333288
LENGTH 6 8 0.769985714153
LENGTH 6 12 1.74652340379
ANGLE 0 3 4 109.474494925
ANGLE 0 3 5 109.508735057
ANGLE 0 3 9 109.484840265
ANGLE 0 6 7 104.089727514
ANGLE 0 6 8 108.925240921
ANGLE 0 6 12 111.993655092
ANGLE 1 0 2 109.493753532
ANGLE 1 0 3 109.46543457
ANGLE 1 0 6 109.540602962
ANGLE 2 0 3 109.405323301
ANGLE 2 0 6 109.482327668
ANGLE 3 0 6 109.439782433
ANGLE 3 9 10 104.171367515
ANGLE 3 9 11 108.77863304
ANGLE 3 9 12 112.511207986
ANGLE 4 3 5 109.471010336
ANGLE 4 3 9 109.375729819
ANGLE 5 3 9 109.512432486
ANGLE 6 12 9 95.0804682662
ANGLE 6 12 13 112.34353709
ANGLE 6 12 14 112.259049919
ANGLE 7 6 8 109.47701199
ANGLE 7 6 12 113.639592149
ANGLE 8 6 12 108.576542027
ANGLE 9 12 13 113.5222725
ANGLE 9 12 14 113.67111377
ANGLE 10 9 11 109.526163306
ANGLE 10 9 12 112.516865618
ANGLE 11 9 12 109.182660523
ANGLE 13 12 14 109.419326451
