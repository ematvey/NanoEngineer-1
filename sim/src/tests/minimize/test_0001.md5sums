stdout 418651447C50AFC9561D3C3A21181D2B
stderr D41D8CD98F00B204E9800998ECF8427E
