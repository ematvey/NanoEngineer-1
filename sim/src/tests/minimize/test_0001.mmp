mmpformat 080327 required; 080523 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (0.936981, 0.308166, -0.163947, 0.014883) (5.314410) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.936981, 0.308166, -0.163947, 0.014883) (4.504046) (0.091500, -1.694500, -0.145000) (1.000000)
egroup (View Data)
group (test_0001)
info opengroup open = True
mol (Chunk-C1) def
atom 1 (6) (-192, 2912, 0) def
atom 2 (1) (237, 3705, 598) def
bond1 1
atom 3 (1) (-791, 3341, -793) def
bond1 1
atom 4 (6) (-1081, 2023, 889) def
bond1 1
atom 5 (1) (-2085, 1999, 486) def
bond1 4
atom 6 (1) (-1105, 2425, 1893) def
bond1 4
atom 7 (6) (935, 2061, -611) def
bond1 1
atom 8 (1) (713, 2051, -1670) def
bond1 7
atom 9 (1) (1881, 2560, -445) def
bond1 7
atom 10 (6) (-509, 593, 921) def
bond1 4
atom 11 (1) (-1279, -11, 458) def
bond1 10
atom 12 (1) (-384, 291, 1952) def
bond1 10
atom 13 (6) (1013, 469, 103) def
bond1 7 10
atom 14 (1) (1856, 359, 773) def
bond1 13
atom 15 (1) (1043, -314, -643) def
bond1 13
egroup (test_0001)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part test_0001
