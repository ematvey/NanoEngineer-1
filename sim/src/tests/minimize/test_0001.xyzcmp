15
RMS=0.972906
C -0.197603 2.906413 -0.024164
H 0.181203 3.763749 0.531738
H -0.784008 3.296541 -0.855719
C -1.082687 2.012268 0.892950
H -2.101095 1.963359 0.508234
H -1.138337 2.434082 1.896206
C 0.981308 2.036665 -0.551106
H 0.965274 1.994101 -1.639900
H 1.936345 2.473963 -0.260874
C -0.448906 0.590833 0.935485
H -1.155835 -0.151858 0.566430
H -0.199778 0.315721 1.960087
C 0.831893 0.608485 0.050500
H 1.708490 0.352962 0.645331
H 0.755737 -0.133283 -0.744197
