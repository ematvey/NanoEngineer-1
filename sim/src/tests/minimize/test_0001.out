======= test_0001.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 6
== structure comparison ==
cutover to fine tolerance at 1
